recipes/pytest-selenium/meta.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: pytest-selenium
+  version: 1.6.0
+
+source:
+  git_rev: v1.6.0
+  git_url: https://github.com/pytest-dev/pytest-selenium.git
+
+build:
+  noarch_python: True
+  number: 1
```

Comment:
For recipes which are basically a `setup.py install` script, I suggest you to use [build/script](http://conda.pydata.org/docs/building/meta-yaml.html#script) instead and ditch the `.bat` and `.sh` files.

```yml
build:
  number: 0
  script: python setup.py install --single-version-externally-managed --record=record.txt
```

Here's another example: https://github.com/conda-forge/pytest-mock-feedstock/blob/master/recipe/meta.yaml#L17

Also, the first packages should start with `build number` `0`.